internal/cmd/load-balancer/target-pool/describe/describe.go

```diff
@@ -0,0 +1,189 @@
+package describe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/load-balancer/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/load-balancer/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/loadbalancer"
+)
+
+const (
+	targetPoolNameArg = "TARGET_POOL_NAME"
+
+	lbNameFlag = "lb-name"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	TargetPoolName string
+	LBName         string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("describe %s", targetPoolNameArg),
+		Short: "Shows details of a target pool in a Load Balancer",
+		Long:  "Shows details of a target pool in a Load Balancer.",
+		Args:  args.SingleArg(targetPoolNameArg, nil),
+		Example: examples.Build(
+			examples.NewExample(
+				`Get details of a target pool with name "pool" in load balancer with name "my-load-balancer"`,
+				"$ stackit load-balancer target-pool describe pool --lb-name my-load-balancer"),
+			examples.NewExample(
+				`Get details of a target pool with name "pool" in load balancer with name "my-load-balancer in JSON output"`,
+				"$ stackit load-balancer target-pool describe pool --lb-name my-load-balancer --output-format json"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd, args)
+			if err != nil {
+				return err
+			}
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("read load balancer: %w", err)
+			}
+
```

Comment:
I would also call `FindLoadBalancerTargetPoolByName` here to get the target pool, and then pass it to output result only if it is not `nil`

Reply:
Great point, I generally agree but it's probably better to do it inside the outputResult, otherwise the arguments for the function are repetive, since we would have: `outputResult(printer, model, targetPool, listener, loadBalancer)`

But I am also open to this way

Reply:
Hm true, and there's the question regarding the JSON response again

But we don't need to return the whole LB in the response, in this case we could return the target pool object with a listener field inside. I think you have already done something similar for some other command, right?

Reply:
I also think it's more clear in this case than just returning the whole LB, which is not what the user is expecting if we just wants to check a single target pool we identified by name

Reply:
Also a good point. I agree, implemented it now